integration/images.go

```diff
@@ -152,7 +152,7 @@ func (d *DockerFileBuilder) BuildImage(imageRepo, gcsBucket, dockerfilesPath, do
 		append([]string{"build",
 			"-t", dockerImage,
 			"-f", path.Join(dockerfilesPath, dockerfile),
-			"."},
+			"../"},
```

Comment:
This should be just `.` is all cases except for the gitRepoTests, in which case it should be the GitHub URL